### ISSUE TITLE
Cr4shed not showing any logs on ios 15.3.1

### ISSUE THREAD

**Issue by foxfortmobile** [Muirey03/Cr4shed#77]
On tethered Palera1n jailbreak, when springboard crashes no logs is shown in Cr4shed app.
Cr4shed version: 4.2.2
Device: iPhone 7
ios: 15.3.1

I put an `abort()` in code running into springboard to force a crash but no logs ever show up in Cr4shed app.

**Comment by Muirey03** [comment]
Can you also try raising an NSException?

**Comment by foxfortmobile** [comment]
yes, throwing exceptions is caught properly and displayed in the app.

**Comment by Muirey03** [comment]
That's good to know, it's just the cr4shedmach code that's now broken. @NSExceptional do you have a palera1n-compatible device?

**Comment by Muirey03** [comment]
I'm afraid I also don't have a device I can jailbreak with palera1n, so I'm not sure we can do much about this. I can look to see if it's just something simple like method renaming on the CrashReport class, but I fear it will be a more breaking change than that.
